<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InserirRetangulo</class>
 <widget class="QDialog" name="InserirRetangulo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Inserir Retângulo</string>
  </property>
  <widget class="QLabel" name="Retangulo">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>10</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Retângulo</string>
   </property>
  </widget>
  <widget class="QLabel" name="Coordenada_x">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Coordenada x:</string>
   </property>
  </widget>
  <widget class="QLabel" name="Coordenada_y">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>60</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Coordenada y:</string>
   </property>
  </widget>
  <widget class="QLabel" name="Cor">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Cor:</string>
   </property>
  </widget>
  <widget class="QLabel" name="Base">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Base:</string>
   </property>
  </widget>
  <widget class="QLabel" name="Altura">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Altura:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Ok">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>250</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Cancel">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>250</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_x">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>60</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_y">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>60</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_base">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>150</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>20</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_altura">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>190</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>20</number>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_cor">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>110</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
